controller/array_action/errors.py

```diff
@@ -99,6 +99,12 @@ def __init__(self, pool, array):
         self.message = messages.PoolDoesNotExist_message.format(pool, array)
 
 
+class PoolIsRequired(BaseArrayActionException):
```

Comment:
this sounds more like a fact rather than an error.
consider renaming to something like `PoolParameterIsMissing/NotSpecified`

Reply:
done